### PR TITLE
Add new release notes for 8Beta2

### PR DESCRIPTION
**Pull request by Mac-Rae**
Pushes new release notes and fixes the wrong type of notices used in 2 locations

### CODE REVIEW

File: content/8.x/admin/releases/8.0-beta/_index.en.adoc [hunk 1]
```diff
@@ -101,4 +101,80 @@ Please link:https://suitecrm.com/download[visit the official website] to find th
 
 To report any security issues please follow our Security Process and send them directly to us via email security@suitecrm.com
 
+'''
+
+== 8.0-beta-2
```
Comment (mattlorimer) [inline content/8.x/admin/releases/8.0-beta/_index.en.adoc:106]: beta 2 should be above beta one and be at the top

**Review by mattlorimer — CHANGES_REQUESTED** [review]

File: content/8.x/admin/releases/8.0-beta/_index.en.adoc (hunk 1 above)
Comment (Mac-Rae) [reply content/8.x/admin/releases/8.0-beta/_index.en.adoc:106]: Thanks! Looked past that one

Comment (Mac-Rae) [reply content/8.x/admin/releases/8.0-beta/_index.en.adoc]: Fixed now

**Comment by Mac-Rae** [comment]
Will be adding punctuation to end of all added release notes, please stand by for another push

**Comment by Mac-Rae** [comment]
Good for review now

**Pull request merged**